tests/Command/RunCommandTest.php

```diff
@@ -237,7 +221,7 @@ public function testExecuteSuccessWithLocalInputOutputInResult(): void
                     'input' => [
                         'tables' => [
                             [
-                                'source' => 'in.c-executor-test.source',
```

Comment:
Tohle byl nesmysl. Spoléhalo to na to že předchozí test ten bucket založil. Upravil jsem to aby to všechno bralo z bucketu pro testy a ta tabulka se na začátku testu vytvořila.